ios/Classes/FlutterLockerPlugin.swift

```diff
@@ -0,0 +1,58 @@
+import Flutter
+import UIKit
+import Locker
+
+public class FlutterLockerPlugin: NSObject, PigeonApi, FlutterPlugin {
+    public func canAuthenticate(completion: @escaping (NSNumber?, FlutterError?) -> Void) {
+        let supportedBiometrics = Locker.configuredBiometricsAuthentication
+        completion(supportedBiometrics != BiometricsType.none ? 1 : 0, nil)
```

Comment:
Have you tried running it on iOS? On my simulator there's exception when calling canAuthenticate:  Unhandled Exception: type 'int' is not a subtype of type 'bool?' in type cast

The issue is that you are returning int here instead of bool. Is there any specific reason you went with ints?